examples/pipeline_wavernn/wavernn.py

```diff
@@ -0,0 +1,586 @@
+        type=int,
+        metavar="N",
+        help="number of data loading workers",
+    )
+    parser.add_argument(
+        "--checkpoint",
+        default="checkpoint.pth.tar",
+        type=str,
+        metavar="FILE",
+        help="filename to latest checkpoint",
+    )
+    parser.add_argument(
+        "--epochs",
+        default=3000,
+        type=int,
+        metavar="N",
+        help="number of total epochs to run",
+    )
+    parser.add_argument(
+        "--start-epoch",
+        default=0,
+        type=int,
+        metavar="N",
+        help="manual epoch number"
+    )
+    parser.add_argument(
+        "--print-freq",
+        default=100,
+        type=int,
+        metavar="N",
+        help="print frequency in epochs",
+    )
+    parser.add_argument(
+        "--batch-size",
+        default=256,
+        type=int,
+        metavar="N",
+        help="mini-batch size"
+    )
+    parser.add_argument(
+        "--learning-rate",
+        default=1e-4,
+        type=float,
+        metavar="LR",
+        help="initial learning rate",
+    )
+    parser.add_argument(
+        "--weight-decay",
+        default=0.0,
+        type=float,
+        metavar="W",
+        help="weight decay"
+    )
+    parser.add_argument(
+        "--adam-beta1",
+        default=0.9,
+        type=float,
+        metavar="AD1",
+        help="adam_beta1"
+    )
+    parser.add_argument(
+        "--adam-beta2",
+        default=0.999,
+        type=float,
+        metavar="AD2",
+        help="adam_beta2"
+    )
+    parser.add_argument(
+        "--eps",
+        metavar="EPS",
+        type=float,
+        default=1e-8
+    )
+    parser.add_argument(
+        "--clip-norm",
+        metavar="NORM",
+        type=float,
+        default=4.0
+    )
+    parser.add_argument(
+        "--scheduler",
+        metavar="S",
+        default="exponential",
+        choices=["exponential", "reduceonplateau"],
+        help="optimizer to use",
+    )
+    parser.add_argument(
+        "--gamma",
+        default=0.999,
+        type=float,
+        metavar="GAMMA",
+        help="learning rate exponential decay constant",
+    )
+    parser.add_argument(
+        "--seed",
+        type=int,
+        default=1000,
+        help="random seed"
+    )
+    parser.add_argument(
+        "--progress-bar",
+        default=False,
+        action="store_true",
+        help="use progress bar while training"
+    )
+    parser.add_argument(
+        "--mulaw",
+        default=True,
+        action="store_true",
+        help="if used, waveform is mulaw encoded"
+    )
+    parser.add_argument(
+        "--jit",
+        default=False,
+        action="store_true",
+        help="if used, model is jitted"
+    )
+    parser.add_argument(
+        '--resume',
+        default='',
+        type=str,
+        metavar='PATH',
+        help='path to latest checkpoint'
+    )
+    # the product of `upsample_scales` must equal `hop_length`
```

Comment:
intentional?

Reply:
did you mean to add this to the help of the command line?

Reply:
Since this has been mentioned in the WaveRNN model, I remove this comment.